trappy/ftrace.py

```diff
@@ -465,6 +482,29 @@ def __parse_trace_file(self, trace_file):
             raise ValueError('Failed to parse ftrace file {}:\n{}'.format(
                 trace_file, str(e)))
 
+    def __add_warnings(self):
```

Comment:
Instead of actually adding attributes and creating a placeholder class, implementing `__getattr__` and raising an exception is enough. `__getattr__` is used at the end of the usual lookup sequence, so it will not impact the time it takes to access other attributes.